### PR TITLE
Provide a fib implementation for jlouis

### PR DESCRIPTION
**Pull request by jlouis**
Most of the entry is pure build-scaffolding. The meat is in a single file, which is then referenced in people.json.

I think I got it mostly right.

- [X] I added myself to [`people.json`](./people.json)!

### CODE REVIEW

File: entries/jlouis/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Fib like only I would write it
+
+## Instructions
+
+Everything is in `bin/fib.ml` and the rest is just scaffolding.
```
Comment (jlouis) [inline entries/jlouis/README.md]: Oh, this requires a fixup

File: people.json [hunk 2]
```diff
@@ -88,28 +99,27 @@
                 "name": "vintage-htdp",
                 "link": "./entries/jyoo980/vintage-htdp/fib.rkt"
             }
-
         ]
     },
-	{
```
Comment (jlouis) [inline people.json:94]: Editor decided to clean up in this file. Could remove this hunk if you so prefer.

Comment (braxtonhall) [reply people.json:94]: LOL no this is good. i should probably make my github action do this automatically

**Comment by braxtonhall** [comment]
Let me know if you think it's done and I'll merge it!

I do have a couple questions though:
1. is the `.opam` file supposed to be empty? (i know nothing about ocaml)
2. where did i mess up to get this out of `dune test`?
```
[nix-shell:~/Documents/fib/entries/jlouis]$ dune test
/nix/store/a5ac77k6289sgbq5izv66djgsvabzhc4-cctools-port-973.0.1/libexec/as/x86_64/as: this system assembler is deprecated. Please migrate to the clang integrated assembler (`as -q').
/nix/store/a5ac77k6289sgbq5izv66djgsvabzhc4-cctools-port-973.0.1/libexec/as/x86_64/as: this system assembler is deprecated. Please migrate to the clang integrated assembler (`as -q').
```

**Comment by jlouis** [comment]
The empty .opam roughly marks this as a module you could push through the opam package manager if you so wanted.

I have no idea why it is trying to use that assembler. Presume this is on MacOS? Haven't done lots of Mac stuff for the last couple of years, so I'm not up to speed where things might go on a tangent.

I'm more or less done. Not perfect, but nothing really is.

**Pull request merged**